### PR TITLE
Add Style Guide for keyboard key

### PR DESCRIPTION
**Pull request by rgaiacs**
https://github.com/swcarpentry/styles/pull/165 requests a style guide. This pull request provide the style guide. **Comments are welcome**.

### CODE REVIEW

File: _episodes/06-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+|-------------------+--------------------------------------------------------------------+--------------------------|
+|    Right arrow    |               Use Unicode "Rightwards arrow" (8594).               |      `<kbd>→</kbd>`      |
+|-------------------+--------------------------------------------------------------------+--------------------------|
+|     Up arrow      |                Use Unicode "Upwards arrow" (8593).                 |      `<kbd>↑</kbd>`      |
+|-------------------+--------------------------------------------------------------------+--------------------------|
+
+## Keyboard Key Combination
+
+When make reference to a keyboard key combination that the reader should press,
+insert a plus sign **withotu sapce** between each one of the keys.
```
Comment (fmichonneau) [inline _episodes/06-style-guide.md]: without

**Comment by fmichonneau** [comment]
+1

File: _episodes/06-style-guide.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+title: "Style Guide"
+teaching: 10
+exercises: 0
+questions:
+- "How are keyboard key combination write?"
```
Comment (clintonroy) [inline _episodes/06-style-guide.md]: Written instead of write, I think.

Comment (willingc) [reply _episodes/06-style-guide.md]: @rgaiacs, Perhaps adjust grammar to:
`How are keyboard key combinations written? or How is a keyboard key combination written?`

File: _episodes/06-style-guide.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+objectives:
+- "Explain keyboard key combination."
+keypoints:
+- "Keyboard keys need to use `<kbd>` HTML tag."
+---
+
+## Keyboard Key
+
+When make reference to a keyboard key that the reader should press
```
Comment (willingc) [inline _episodes/06-style-guide.md]: When making a reference...

File: _episodes/06-style-guide.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+the HTML tag `<kbd>` **must** be used to enclose the key label.
+For example, "to delete the cell press <kbd>D</kbd>" should be write as
+
+~~~
+to delete the cell press <kbd>D</kbd>
+~~~
+
+The table below cover most of the keyboard key labels.
```
Comment (willingc) [inline _episodes/06-style-guide.md]: ~~cover~~ covers

File: _episodes/06-style-guide.md (hunk 1 above)
Comment (willingc) [reply _episodes/06-style-guide.md]: When making reference to ...

**Pull request merged**